pyiceberg/table/inspect.py

```diff
@@ -57,7 +87,21 @@ def _get_snapshot(self, snapshot_id: Optional[int] = None) -> Snapshot:
             raise ValueError("Cannot get a snapshot as the table does not have any.")
 
     def snapshots(self) -> "pa.Table":
-        import pyarrow as pa
```

Comment:
i think we actually need this here, in case someone imports this function directly
```
from pyiceberg.table.inspect import snapshots
```